### PR TITLE
[WFLY-9336] Extend Elytron security context propagation test coverage

### PR DESCRIPTION
**Pull request by olukas**
https://issues.jboss.org/browse/WFLY-9336

This PR replaces original https://github.com/wildfly/wildfly/pull/10530

How it differs from original PR:
* New test cases for WFCORE-3267 have been implemented
* Use upstream jira in `@Ignore`
* Removed test case for JBEAP-13217 since it is unsupported scenario

### CODE REVIEW

**Comment by olukas** [comment]
Retest this please

**Comment by kabir** [comment]
Retest this please

File: testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/elytron/seccontext/ServerChainSecurityContextPropagationTestCase.java [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.wildfly.test.manual.elytron.seccontext;
+
+import java.io.IOException;
```
Comment (darranl) [inline testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/elytron/seccontext/ServerChainSecurityContextPropagationTestCase.java]: Probably not enough to block merging but the import ordering is incorrect, statics go first.

**Review by darranl — APPROVED** [review]
Overall looks good, did spot one class with badly ordered imports but that probably should not be sufficient to block merging.

**Comment by kabir** [comment]
Retest this please

**Comment by olukas** [comment]
Thanks for review @darranl. I've fixed the import order in FirstServerChainBean and ServerChainSecurityContextPropagationTestCase.

**Pull request merged**